Tests/AK/TestHashFunctions.cpp

```diff
@@ -53,3 +54,53 @@ TEST_CASE(constexpr_ptr_hash)
     // "ptr_hash" test binds the result.
     static_assert(ptr_hash(FlatPtr(42)));
 }
+
+// Testing concrete hash results is not possible due to SipHash's non-determinism.
+// We instead perform some sanity checks and try to hit any asserts caused by programming errors.
+TEST_CASE(sip_hash)
+{
+    EXPECT_EQ(standard_sip_hash(42), standard_sip_hash(42));
+    EXPECT_EQ(secure_sip_hash(42), secure_sip_hash(42));
+    EXPECT_NE(standard_sip_hash(42), secure_sip_hash(42));
```

Comment:
@gmta asked on discord, and I'll ask here again: is this guaranteed to not produce the same value? or just very unlikely?

Reply:
The likelihood is 2^{-64}, so I think we win the lottery several million times first.

Reply:
:thinking: I doubt that figure, but sure.